ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -361,7 +362,7 @@ private void collectCommitInformation(TezWork work) throws IOException, TezExcep
           .filter(name -> name.endsWith("HiveIcebergNoJobCommitter")).isPresent();
       // we should only consider jobs with Iceberg output committer and a data sink
       if (hasIcebergCommitter && !vertex.getDataSinks().isEmpty()) {
-        String tableLocationRoot = jobConf.get("location");
+        String tableLocationRoot = jobConf.get(ICEBERG_MR_TABLE_LOCATION);
```

Comment:
Why it is `mr`?

Reply:
My mistake. It was the intellij-generated constant, and didn't realize it had MR in it :)